go/adbc/driver/internal/driverbase/driver_info.go

```diff
@@ -43,6 +45,39 @@ var infoValueTypeCodeForInfoCode = map[adbc.InfoCode]adbc.InfoValueTypeCode{
 	adbc.InfoVendorSubstraitMaxVersion: adbc.InfoValueStringType,
 }
 
+const (
+	// namespace prefix
+	otelInfoSemConv = "apache.arrow.adbc.info."
```

Comment:
should we use https://github.com/open-telemetry/opentelemetry-go/blob/v1.36.0/attribute/key.go#L8 instead of a raw string to have the extra methods for easier usage?

Reply:
Now using `attribute.Key`